### ISSUE TITLE
omf install completions doesn't show themes

### ISSUE THREAD

**Issue by derekstavis** [oh-my-fish/oh-my-fish#251]
Currently install completions does not show available themes.

**Comment by decors** [comment]
how about like this?

```
function __omf_install_completion
  for theme in (omf.packages.list --available --theme)
    printf "%s\ttheme\n" $theme
  end
  for plugins in (omf.packages.list --available --plugins)
    printf "%s\tplugins\n" $plugins
  end
end

complete -c omf -f -n "__fish_seen_subcommand_from i install" -a "(__omf_install_completion)"
```

**Comment by oranja** [comment]
This seems to do  a great start, but can you explain to someone who never did completions before (me), why there's a need to print "theme" after each theme name, and "plugins" after each plugin name?

In other words, why can't this simply be:
```fish
complete -c omf -f -n "__fish_seen_subcommand_from i install" -a "(omf.packages.list --available)"
```
?

**Comment by decors** [comment]
@oranja Thanks

My code has some errors

```
function __omf_install_completion
  for t in (omf.packages.list --available --theme)
    printf "%s\ttheme\n" $t
  end
  for p in (omf.packages.list --available --plugins)
    printf "%s\tplugin\n" $p
  end
end

complete -c omf -f -n "__fish_seen_subcommand_from i install" -a "(__omf_install_completion)"
```

```
> omf install 
agnoster             (theme)
agnoster-mercurial  (plugin)
ansible                     
apt                 (plugin)
artisan                     
balias              (plugin)
bang-bang           (plugin)
```

**Comment by oranja** [comment]
Oh! Now I see why you put "theme"/"plugin" after each option. That actually gives a great user experience in my opinion. :)

I don't see any error in your code now. Run `complete -e -c omf -f -n "__fish_seen_subcommand_from i install"` before your code to erase all current completions for `omf install`. You'll see that your code produces the desired results.

I noticed, however, that breaking down the completion into smaller commands results in a more responsive shell. Can someone else confirm that It's not my imagination?

```fish
complete -e -c omf -f -n "__fish_seen_subcommand_from i install"
```
```fish
for t in (omf.packages.list --available --theme)
  complete -c omf -f -n "__fish_seen_subcommand_from i install" -a "(printf '%s\ttheme' $t)"
end
for p in (omf.packages.list --available --package)
  complete -c omf -f -n "__fish_seen_subcommand_from i install" -a "(printf '%s\tpackage' $p)"
end
```

**Comment by coderstephen** [comment]
Fixed by #263.
